src/locales/en.json

```diff
@@ -96,6 +96,10 @@
     "title": "Games"
   },
   "games": {
+    "quizgame": {
+      "title": "Quiz Game",
```

Comment:
I think we should also be adding these to the ar.json and tr.json files

Reply:
PTAL